src/vendor/errors.jsx

```diff
@@ -1,154 +1,255 @@
 /* eslint-disable react/no-multi-comp */
+/* eslint-disable max-len */
 import React from 'react';
-import PropTypes from 'prop-types';
-import { missing } from './utils';
+import ErrorPanel from '@mozilla-frontend-infra/components/ErrorPanel';
+import { coalesce, exists, isChrome, isNode, isString, missing } from './utils';
 import { expand } from './Template';
-
-const newIssue =
-  'https://github.com/mozilla/firefox-health-dashboard/issues/new';
-const BasicError = ({ error }) => {
-  if (error.toString)
-    return <pre style={{ fontSize: '1.0rem' }}>{error.toString()}</pre>;
-
-  if (error.template)
-    return <pre style={{ fontSize: '1.0rem' }}>{error.template}</pre>;
-
-  if (error.cause && error.cause.message)
-    return <pre style={{ fontSize: '1.0rem' }}>{error.cause.message}</pre>;
-
-  return (
-    <p style={{ fontSize: '1.0rem' }}>
-      <span>
-        There has been a critical error. We have reported it. If the issue is
-        not fixed within few hours please file an issue:
-        <br />
-        <a href={newIssue} target="_blank" rel="noopener noreferrer">
-          {newIssue}
-        </a>
-      </span>
-    </p>
+import { frum } from './queryOps';
+
+let stackPatterns = [];
+
+if (isNode) {
+  //   Error
+  //       at Function.Object.<anonymous>.Log.error (C:\Users\kyle\code\firefox-health-dashboard\src\vendor\errors.jsx:174:9)
+  //       at Object.error (C:\Users\kyle\code\firefox-health-dashboard\test\vendor\errors.test.js:9:16)
+  //       at Object.asyncJestTest (C:\Users\kyle\code\firefox-health-dashboard\node_modules\jest-jasmine2\build\jasmine_async.js:108:37)
+  //       at resolve (C:\Users\kyle\code\firefox-health-dashboard\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
+  //       at new Promise (<anonymous>)
+  //       at mapper (C:\Users\kyle\code\firefox-health-dashboard\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
+  //       at promise.then (C:\Users\kyle\code\firefox-health-dashboard\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
+  //       at process.internalTickCallback (internal/process/next_tick.js:77:7)
```

Comment:
Do you need this commented block?

Reply:
I would like to keep this:  Every regex should have examples of what it matches.